platform/lib/utils/project.js

```diff
@@ -62,6 +62,7 @@ const paths = {
   INLINE_EXAMPLES_DEST: absolute('dist/inline-examples'),
   PAGES_SRC: absolute('pages/content/amp-dev'),
   STATICS_DEST: absolute(`${DIST_DIR}/static`),
+  BENTO_COMPONENTS_LIST: absolute(`${DIST_DIR}/static/bento-components.json`),
```

Comment:
What's the plan with this?

Reply:
We might use it to automate this list here: https://github.com/ampproject/amp.dev/pull/5198/files#diff-4184c78462ed1fab4a0530e0ada531051198fa358ce91308ac1e08ad1fc305d6R206-R222 🙂

Reply:
Hah! Now I remember, that was part of the reason why I added it.